tests/utils.py

```diff
@@ -204,26 +204,26 @@ def __init__(self, caplog, index=0, allow_errors=True, ignore=[]):
         self.allow_errors = allow_errors
         self.ignore = ignore
 
-    def _find(self, loggerpart, level, msg, after=0):
+    def _find(self, loggerpart, level, msg, after=0, min_level=1):
         for i, (logger_name, log_level, message) in enumerate(self.caplog.record_tuples[after:]):
             if msg in message:
-                if loggerpart in logger_name and level == log_level:
+                if loggerpart in logger_name and (level == log_level or (log_level < min_level)):
```

Comment:
Isn't `min_level` a strange name here when doing the comparison `log_level < min_level`. It defines an upper limit. Shouldn't it be `max_level`?

Reply:
this was plain wrong